prow/cluster/08-prow-addons-ctrl-manager.yaml

```diff
@@ -0,0 +1,132 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels:
+    app: prow-addons-ctrl-manager
+    control-plane: controller-manager
+    controller-tools.k8s.io: "1.0"
+  name: prow-addons-controller-manager
+  namespace: default
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  labels:
+    app: prow-addons-ctrl-manager
+  name: prow-addons-manager-role
+rules:
+- apiGroups:
+  - prow.k8s.io
+  resources:
+  - prowjobs
+  verbs:
+  - get
+  - list
+  - watch
+  - create
+  - update
+  - patch
+  - delete
+- apiGroups:
+  - prow.k8s.io
+  resources:
+  - prowjobs/status
+  verbs:
+  - get
+  - update
+  - patch
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  labels:
+    app: prow-addons-ctrl-manager
+  name: prow-addons-manager-rolebinding
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: prow-addons-manager-role
+subjects:
+- kind: ServiceAccount
+  name: prow-addons-controller-manager
+  namespace: default
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: prow-addons-ctrl-manager
+    control-plane: controller-manager
+    controller-tools.k8s.io: "1.0"
+  name: prow-addons-controller-manager-service
+  namespace: default
+spec:
+  ports:
+  - name: http
+    port: 80
+    targetPort: http
+  selector:
+    app: prow-addons-ctrl-manager
+    control-plane: controller-manager
+    controller-tools.k8s.io: "1.0"
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  labels:
+    app: prow-addons-ctrl-manager
+    control-plane: controller-manager
+    controller-tools.k8s.io: "1.0"
+  name: prow-addons-controller-manager
+  namespace: default
+spec:
+  selector:
+    matchLabels:
+      app: prow-addons-ctrl-manager
+      control-plane: controller-manager
+      controller-tools.k8s.io: "1.0"
+  serviceName: prow-addons-controller-manager-service
+  template:
+    metadata:
+      labels:
+        app: prow-addons-ctrl-manager
+        control-plane: controller-manager
+        controller-tools.k8s.io: "1.0"
+    spec:
+      containers:
+      - command:
+        - /manager
+        env:
+        - name: NOTIFIER_GITHUB_ACCESS_TOKEN
+          valueFrom:
+            secretKeyRef:
+              key: oauth
+              name: oauth-token
+        - name: NOTIFIER_SLACK_TOKEN
+          valueFrom:
+            secretKeyRef:
+              key: token
+              name: sap-slack-bot-token
+        - name: NOTIFIER_SLACK_REPORTER_CHANNEL
+          value: '#c4core-kyma-ci-force'
+        - name: NOTIFIER_SLACK_REPORTER_ACT_ON_PROW_JOB_TYPE
+          value: periodic,postsubmit
+        - name: NOTIFIER_SLACK_REPORTER_ACT_ON_PROW_JOB_STATE
+          value: failure,error
+        image: eu.gcr.io/kyma-project/test-infra/develop/prow-addons-ctrl-manager:0.0.1
```

Comment:
will be change after merging this PR: https://github.com/kyma-project/test-infra/pull/592